ec2_instances/config.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+#variables
+SSH_KEY_NAME="poc-aws-key"
+SSH_PRIVATE_FILE_NAME="poc-aws-key.pem"
+SG_NAME="poc-aws"
+SG_TAG_KEY="Name"
+SG_TAG_VAL="poc-aws"
+SG_ID=""
+UBUNTU_X86_ID="ami-0caef02b518350c8b"
+EC2_TYPE="t2.micro"
+EC2_TAG_KEY="Name"
```

Comment:
Would suggest to use the same key tag/value for all recourses.  It would be easier to search resources by the tag for the decommission process.